src/github.ts

```diff
@@ -495,3 +531,24 @@ export async function getContent(path: string): Promise<string | undefined> {
 export async function currentHeadMatchesTag(tagName: string): Promise<boolean> {
   return (await getShaForTag(tagName)) === github.context.sha;
 }
+
+/**
+ * Returns the commits between two Git refs.
+ * According to the docs, the behavior matches `baseRef...compRef`, with an additional
+ * "chronological" order (unsure if commit- or author-date).
+ *
+ * Returns a list ICommits representing `baseRef...compRef`.
+ */
+export async function getCommitsBetweenRefs(
+  baseRef: string,
+  compRef?: string
+): Promise<ICommit[]> {
+  const { data: resp } =
+    await getOctokit().rest.repos.compareCommitsWithBasehead({
+      owner: OWNER,
```

Comment:
General improvement, we can replace all the following params;
```
{
  owner: OWNER,
  repo: REPO
 }
```

with
```
{
  ...github.context.repo,
}
```

Reply:
I'll add a refactor commit  to replace that in this file completely, otherwise we'll keep copy-pasting the current form ;)